pkg/blob/controllerserver_test.go

```diff
@@ -893,20 +881,6 @@ func TestDeleteVolume(t *testing.T) {
 				}
 			},
 		},
-		{
```

Comment:
This case is deleted because fake driver has got required capabilities.

Reply:
what's the result of this test now? `invalid create volume req"` ?

Reply:
original test is for testing `invalid create volume req`, not capabilities